api/types.proto

```diff
@@ -1156,6 +1156,33 @@ message Privileges {
 		string level = 5;
 	}
 	SELinuxContext selinux_context = 2 [(gogoproto.customname) = "SELinuxContext"];
+
+	message SeccompOpts {
```

Comment:
Let's add a doc-comment describing the field, and maybe linking to the Docker seccomp docs